### PR TITLE
Item element for keywords

### LINKED ISSUE #104: Add possibility to use item object instead of locator

### PR DESCRIPTION
**Pull request by eeromakiesko**
Item related keywords like Click Button, Click Item can now be used with item reference i.e.
    ${item}=    Get Item    mybutton
    Click Button    ${item}

Closes #104

### CODE REVIEW

File: src/WhiteLibrary/__init__.py [hunk 1]
```diff
@@ -132,12 +143,20 @@ def __init__(self):
         DynamicCore.__init__(self, self.libraries)
 
     def _get_typed_item_by_locator(self, item_type, locator):
-        search_criteria = self._get_search_criteria(locator)
-        return self.window.Get[item_type](search_criteria)
+        if isinstance(locator, UIItem):
+            if not isinstance(locator, item_type):
+                raise TypeError('Assumed that locator item is of type item_type')
+            return locator
+        else:
```
Comment (rasjani) [inline src/WhiteLibrary/__init__.py]: Unnecessary else here, isinstance() branch will either throw or return already at this stage so. Also, its not pythonic :D  Same in _get_item_by_locator()

Comment (eeter) [reply src/WhiteLibrary/__init__.py]: Not sure if we should prevent using else after return..? While it's unnecessary, IMO the code still looks readable either way (there seem to be varying views on which is better). pylint enforces the rule though, so either the else statement should be removed or no-else-return should be disabled.

Comment (rasjani) [reply src/WhiteLibrary/__init__.py]: matter of taste and coding style .. this is essentially a check in pylint and pr build was complaining about that. I'll remove the check if its not wanted/needed.

Comment (eeter) [reply src/WhiteLibrary/__init__.py]: Had a brief discussion about this, let's keep the pylint check for consistency :)

**Pull request merged**